src/Question.jsx

```diff
@@ -7,6 +7,8 @@ import Divider from '@material-ui/core/Divider';
 import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 
+export let gameTimer = setTimeout(0);
```

Comment:
**Request:** As mentioned by _eslint_, it can be really bad to export mutable bindings. This `gameTimer` really doesn't need to be exported either. Have a look at [`componentWillUnmount`](https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class) and remove the timer there instead.